tests/benchmarks/current_consumption/system_off/prj.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_PM_DEVICE=y
+CONFIG_GPIO=y
+CONFIG_CRC=y
```

Comment:
CRC is from original sample?

Reply:
yes; https://github.com/nrfconnect/sdk-zephyr/blob/main/samples/boards/nrf/system_off/prj.conf#L3